tools/src/main/java/org/apache/kafka/tools/MetadataQuorumCommand.java

```diff
@@ -136,17 +143,21 @@ private static void addDescribeParser(Subparsers subparsers) {
             .addArgument("--replication")
             .help("Detailed information about the status of replication")
             .action(Arguments.storeTrue());
+        replicationArgs
+            .addArgument("-hr", "--human-readable")
```

Comment:
This doesn't work without `--replication`, right? In which case, this should probably be a subParser of replication (similar to how we handle subparsing for --describe)

Reply:
It doesn't work with --status, but I see what you mean. We should raise an error, rather than silently ignore the --hr option.